src/pages/home/report/ReportActionCompose.js

```diff
@@ -407,6 +418,12 @@ class ReportActionCompose extends React.Component {
         if (newComment) {
             this.debouncedBroadcastUserIsTyping();
         }
+
+        const hasExceededMaxCommentLength = this.comment.length > CONST.MAX_COMMENT_LENGTH;
+        const exceededCommentLength = hasExceededMaxCommentLength ? this.comment.length : undefined;
+        if (this.state.exceededCommentLength !== exceededCommentLength) {
```

Comment:
I think we can remove this check now as this is Pure Component.

Reply:
good catch, updated!